workspaces/jenkins/plugins/jenkins-backend/README.md

```diff
@@ -22,73 +22,6 @@ In your `packages/backend/src/index.ts` make the following changes:
   backend.start();
 ```
 
-## Integrating into a backstage instance
```

Comment:
I would use this as the title for line 11 above and then remove line 13, you'll also want to add some details about running the yarn add command, you could just copy that from lines 29-32 below.